integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -193,10 +193,9 @@
                         }
                     },
 					{
-						bidder: 'sekindo',
+						bidder: 'sekindoUM',

```

Comment:
Please use spaces rather than tabs in this section to match the rest of the file